ocaml/idl/datamodel.ml

```diff
@@ -9809,6 +9821,178 @@ module SDN_controller = struct
 end
 let sdn_controller = SDN_controller.obj
 
+module PUSB = struct
+  let lifecycle = [Published, rel_inverness, ""]
+
+  let scan = call
+      ~name:"scan"
+      ~lifecycle
+      ~params:[]
+      ~allowed_roles:_R_POOL_OP
+      ()
+
+  let obj =
+    create_obj
+      ~name: _pusb
+      ~descr:"Describes the physical usb device"
```

Comment:
Minor point, but I think `A physical USB device` would be better.

Reply:
updated